src/trie/algo.go

```diff
@@ -0,0 +1,12 @@
+package trie
+
+func (t *trie) findAllStringsWithDFS(currentNode *trieNode, matchingStrings *[]string, currentString string) {
```

Comment:
give some reasonable name .. showing which algorithm we are using does not look better
or we can do .. "genererateAllPossibleStringWithCurrentPrefix" which calls "DFSHelper"